### PR TITLE
Updated docs for Fusion customizations

### LINKED ISSUE #3761: As an organization I can customise the landing page background to fit my usecase
## Description
The default image chosen for the landing page is neuroscience specific. While it may not bother an external user, we should allow for configuration of this image so they can customise it to their own usecase/organization.
This should include update to documentation on how to configure this and give graphical guidelines considering the white fonts that overlap the image.

### PR DESCRIPTION
**Pull request by danburonline**
Fixes #3761

This PR introduces markdown modifications for customizing the Fusion page through environment variables. I added example screenshots to the documentation for better visualization of customization options. Furthermore, I corrected some typos in the file I worked on. These are the additional docs adjustments for the PR in the Fusion repository. You can find the PR [here](https://github.com/BlueBrain/nexus-web/pull/1437).

Here's an example screenshot:

example

### CODE REVIEW

**Review by imsdu — APPROVED** [review]

File: docs/src/main/paradox/docs/fusion/architecture.md [hunk 1]
```diff
@@ -2,30 +2,32 @@
...
-We produce a docker image that is able to serve the compiled assets directly, which is available on
+We produce a Docker image that is able to serve the compiled assets directly, which is available on
 @link:[Dockerhub](https://hub.docker.com/repository/docker/bluebrain/nexus-web){ open=new }.
 
 Although the application is served by a Node.js server, the client communicates directly to @ref:[Nexus Delta](../delta/index.md) using @ref:[Nexus.js](../utilities/index.md#nexus-js).
 
 ## Pages
 
-Nexus Fusion has undergone a significant restructuring to transition from a SubApp-based architecture to a page-based structure. This change will prepare the application for a full migration to file-system based routing technologie and help seperating the application main features.
+Nexus Fusion has undergone a significant restructuring to transition from a SubApp-based architecture to a page-based structure. This change will prepare the application for a full migration to file-system based routing system and help separating the application main features.
```
Comment (Dinika) [inline docs/src/main/paradox/docs/fusion/architecture.md:18]: For some reason "...and help **separate** the **application's** main features", reads better to me.

File: docs/src/main/paradox/docs/fusion/architecture.md [hunk 2]
```diff
@@ -2,30 +2,32 @@
...
 
+@@@ note { .tip title="Change of search page" }
+
 The search page has been removed, but every element that was previously available on the search page now has a link on the home page. These links redirect users to the appropriate global search type page.
 
-The pages in Nexus Fusion are part of the source code and reside in the `src/pages` folder.
+@@@
 
-While all the previous feature is still available in the current version of Nexus Fusion, we expect its functionality to evolve and change in the upcoming release.
+The pages in Nexus Fusion are part of the source code and reside in the `src/pages` folder. While all the previous feature is still available in the current version of Nexus Fusion, we expect its functionality to evolve and change in the upcoming releases.
```
Comment (Dinika) [inline docs/src/main/paradox/docs/fusion/architecture.md:30]: "While all the previous **features are** still available..."

Comment (Dinika) [reply docs/src/main/paradox/docs/fusion/architecture.md:30]: "we expect **their** functionality to..."

File: docs/src/main/paradox/docs/fusion/architecture.md [hunk 3]
```diff
@@ -36,16 +38,32 @@ configuration, and the manifest should be hosted somewhere Nexus Fusion can requ
...
+
+![Example organization env](./assets/organizations-envs.png)
+![Example projects env](./assets/projects-envs.png)
+![Example studios env](./assets/studios-envs.png)
+
+- `ORGANIZATION_IMG`: Provide the HTTPS URL for an image on the organization page. @link:[Click here for an example image](https://github.com/BlueBrain/nexus-web/blob/main/src/shared/images/sscx-by-layers-v3.png){ open=new }. The recommended dimensions are approximately 1500 × 450 pixels. Please ensure that the image size is kept under 200 KB.
+- `PROJECTS_IMG`: Share the HTTPS URL for an image on the projects page. @link:[Click here for an example image](https://github.com/BlueBrain/nexus-web/blob/main/src/shared/images/hippocampus.png){ open=new }. The recommended dimensions are approximately 1500 × 450 pixels. Please ensure that the image size is kept under 200 KB.
+- `STUDIOS_IMG`: Provide the HTTPS URL for an image on the studios page. @link:[Click here for an example image](https://github.com/BlueBrain/nexus-web/blob/main/src/shared/images/neocortex.png){ open=new }. The recommended dimensions are approximately 1500 × 450 pixels. Please ensure that the image size is kept under 200 KB.
```
Comment (Dinika) [inline docs/src/main/paradox/docs/fusion/architecture.md:63]: Reading this it's a bit unclear to me if the 200 KB size is a recommendation or mandatory (i.e. if the size exceeds 200 KB, an error will be thrown). If it's a recommendation then maybe we could add somethign along the lines of "Pleas ensure that the image size is kept under 200 KB to provide a load time/user expierience."

**Review by Dinika — COMMENTED** [review]
Looks great! I love the annotations on the images :star2:

File: docs/src/main/paradox/docs/fusion/architecture.md (hunk 3 above)
Comment (danburonline) [reply docs/src/main/paradox/docs/fusion/architecture.md:63]: #4533

File: docs/src/main/paradox/docs/fusion/architecture.md (hunk 1 above)
Comment (danburonline) [reply docs/src/main/paradox/docs/fusion/architecture.md:18]: #4533

File: docs/src/main/paradox/docs/fusion/architecture.md (hunk 2 above)
Comment (danburonline) [reply docs/src/main/paradox/docs/fusion/architecture.md:30]: #4533

**Pull request merged**